src/DropdownMenu.tsx

```diff
@@ -114,7 +159,7 @@ const DropdownMenu: DropdownMenu = React.forwardRef(
       rootCloseEvent,
       show: showProps,
       alignEnd: alignRight,
-      usePopper: !isNavbar,
+      usePopper: !isNavbar && alignClasses.length === 0,
```

Comment:
so we don't want the "flip" behavior when specifying alignment explicitly?

Reply:
Upstream bootstrap uses styles that are incompatible with popper positioning, so they say to disable it.

>If you want to use responsive alignment, disable dynamic positioning by adding the data-display="static" attribute and use the responsive variation classes.

The responsive examples unfortunately do not support flip

Reply:
Added note in a callout:

> Using responsive alignment will disable Popper usage so any dynamic positioning features such as <code>flip</code> will not work.